pkg/model/gcemodel/autoscalinggroup.go

```diff
@@ -137,6 +142,11 @@ func (b *AutoscalingGroupModelBuilder) Build(c *fi.ModelBuilderContext) error {
 
 			case kops.InstanceGroupRoleNode:
 				t.Tags = append(t.Tags, b.GCETagForRole(kops.InstanceGroupRoleNode))
+
+			case kops.InstanceGroupRoleBastion:
+				// TODO: Bastion instances should have public IPs or should they?
```

Comment:
At least on AWS, yes, the bastion instance is a bare machine (no kubelet) which runs SSH but has a public IP.

Reply:
Oh - but for we do sometimes put them behind a load balancer; I suspect they do still need public IPs but I'm not certain!

Reply:
IMO we will need the bastion load balancer to ease managing (and assessing) bastion instance public IP which can change in situations like a respawn.